### PR TITLE
IIS Web app deploy - Added return after reject

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/IISWebAppManagementOnMachineGroupV0/task.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 0,
         "Minor": 5,
-        "Patch": 12
```
Comment (vincentdass) [inline Tasks/IISWebAppManagementOnMachineGroupV0/task.json]: u changed mange task instead of deploy task 😝

Comment (Ajay-MS) [reply Tasks/IISWebAppManagementOnMachineGroupV0/task.json]: Resolved

**Comment by jons-aura** [comment]
I was told that this https://developercommunity.visualstudio.com/content/problem/372169/error-message-on-vsts-release-but-release-appears.html item was fixed by this pull request but I'm still getting the error.

2019-07-01T18:42:12.6426687Z ##[debug]task result: Failed
2019-07-01T18:42:12.6426687Z ##[error]Unhandled: Cannot read property 'value' of null
2019-07-01T18:42:12.6426687Z ##[debug]Processed: ##vso[task.issue type=error;]Unhandled: Cannot read property 'value' of null
2019-07-01T18:42:12.6426687Z ##[debug]Processed: ##vso[task.complete result=Failed;]Unhandled: Cannot read property 'value' of null
2019-07-01T18:42:12.6426687Z ##[debug]Unable to find the location of MS Deploy from registry on machine (Error : ProcessUncleanExitError: QUERY command exited with code 1:

ERROR: The system was unable to find the specified registry key or value.

**Comment by vincentdass** [comment]
@jons-aura , the deployment is in progress and the fix should be available in a week or two.

**Pull request merged**